types/tx/tx.go

```diff
@@ -0,0 +1,68 @@
+package tx
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+func (m *Tx) GetMsgs() []sdk.Msg {
```

Comment:
I'm not sure if any of these methods are needed. The intention was to just use `builder` and not have the proto `Tx` implement `sdk.Tx`